src/Nri/Colors.elm

```diff
@@ -166,6 +167,16 @@ coralLightest =
 
 {-|
 
+<p style="font-size:2em; color: #eef9ff; background-color: black;">#eef9ff</p>
```

Comment:
@stoeffel this ties in to this issue kinda: https://github.com/stoeffel/elm-verify-examples/issues/25